submissions/MosQuitO404/js-dom/index.js

```diff
@@ -0,0 +1,129 @@
+
+    williamsInfo: '<h3>In profile</h3><p>Driven on by the brilliance and passion of the late Sir Frank Williams, Williams grew from humble beginnings to become a Formula 1 behemoth, unrivalled by all except Ferrari and McLaren in terms of enduring success. Over the past four decades the team has racked up Grand Prix wins and championship glory, and in the process nurtured some of the greatest talents in the sport, both in and out the cockpit. Now, following the Williams family`s decision to step aside after the sale of the team to Dorilton Capital, a new era begins...</p>'
+}
+
+// Create teams
+
+class Team {
+    #teamTitle
+    #firstPilotName;
+    #secondPilotName;
+    #firstPilotFlagImage;
+    #secondPilotFlagImage;
+    #firstPilotImage;
+    #secondPilotImage;
+    #teamInfo;
+    #html;
+
+    constructor(teamTitle, firstPilotName, secondPilotName, firstPilotFlagImage, secondPilotFlagImage, firstPilotImage, secondPilotImage, teamInfo) {
+        this.#teamTitle = teamTitle;
+        this.#firstPilotName = firstPilotName;
+        this.#secondPilotName = secondPilotName;
+        this.#firstPilotFlagImage = firstPilotFlagImage;
+        this.#secondPilotFlagImage = secondPilotFlagImage;
+        this.#firstPilotImage = firstPilotImage;
+        this.#secondPilotImage = secondPilotImage;
+        this.#teamInfo = teamInfo;
+        this.#html = `<div class="pilots-container">
+                    <div class="pilot-wrapper">
+                        <span id="first-pilot-name" class="pilot-name"><h3>${this.#firstPilotName}</h3></span>
+                        <span id="first-pilot-flag"><img class="flag-img" src="${this.#firstPilotFlagImage}" alt=""></span>
+                        <img id="first-pilot" class="pilot-img" src="${this.#firstPilotImage}" alt="">
+                    </div>
+                    <div class="pilot-wrapper">
+                        <span id="second-pilot-name" class="pilot-name"><h3>${this.#secondPilotName}</h3></span>
+                        <span id="second-pilot-flag"><img class="flag-img" src="${this.#secondPilotFlagImage}" alt=""></span>
+                        <img id="second-pilot" class="pilot-img" src="${this.#secondPilotImage}" alt="">
+                    </div>
+                    <div class="team-info-container">${this.#teamInfo}</div>
+                </div>`
+    }
+
+    get htmlRender() {
+        const placeholder = document.querySelector('#main-container');
+        const menu = document.querySelector('#menu');
+        
+        menu.addEventListener('click', (e) => {
+            e.preventDefault();
+            if (e.target.dataset.action === this.#teamTitle ) {
+                placeholder.innerHTML = '';
```

Comment:
Try to not recreate the container. When you recreate container your content reflows and the user sees a glitch. It looks like this:
![Screenshot 2022-09-13 at 23 32 38](https://user-images.githubusercontent.com/16616791/190003682-bcb93074-bf15-4d5f-9aab-6a6a6e16ff54.png)

Reply:
Done!